internal/statesync/dispatcher_test.go

```diff
@@ -13,145 +13,104 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/tendermint/tendermint/internal/p2p"
+	"github.com/tendermint/tendermint/internal/test/factory"
 	ssproto "github.com/tendermint/tendermint/proto/tendermint/statesync"
 	"github.com/tendermint/tendermint/types"
 )
 
+var (
+	peer = factory.NodeID("a")
+)
```

Comment:
I'm fine with a global here, but I'd suggest giving it a more contextual name, e.g. `testPeer` or something. That way it's (somewhat more) clear at the usage site that the reader isn't looking at a local to the test. YMMV.
```suggestion
var testPeer = factory.NodeID("a")
```